services/src/contexts/schema.sql

```diff
@@ -88,7 +191,7 @@ CREATE TABLE project_version_layers (
     name character varying(256) NOT NULL,
     workflow_id uuid NOT NULL,
     -- TODO: REFERENCES workflows(id)
-    symbology json,
+    symbology "Symbology",
```

Comment:
what about constraints on the symbology? e.g. exactly one of raster/point/line/polygon should be not null. Also for all the other types we could constrain what fields are nullable and which not. Unfortunately postgres doesn't allow specifying this at the type level, so we would have to do it here. But, how far do we want to go? And what happens if e.g. a symbology has a raster and point symbology? Would we check that in the application? Would we just use the first one?

Reply:
we discussed this in Slack, we should propose "domain wrapping" to rust-postgres and go from there.